src/pages/SignUp/SignUp.scss

```
@@ -0,0 +1,40 @@
+.sign_up {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+  width: 300px;
+  margin-top: 10%;
+
+  .title {
+    font-size: 36px;
+  }
+
+  .input_box {
+    height: 80px;
+    .input_title {
+      font-size: 20px;
+    }
+    .input_warn {
+      font-size: 20px;
+      color: #ff0000;
+    }
+    .input {
+      margin: 0px 10px 10px 0px;
+      height: 50px;
+      padding-left: 10px;
+      width: 300px;
+      border: 1px solid #666;
+      border-radius: 5px;
+    }
+  }
```

```suggestion
  .input_box {
    height: 80px;
    
    .input_title {
      font-size: 20px;
    }
    
    .input_warn {
      font-size: 20px;
      color: #ff0000;
    }
    
    .input {
      margin: 0px 10px 10px 0px;
      height: 50px;
      padding-left: 10px;
      width: 300px;
      border: 1px solid #666;
      border-radius: 5px;
    }
  }
```
가독성을 위해 줄바꿈을 사용해주세요!